OpenRA.Mods.Common/Widgets/ScrollPanelWidget.cs

```diff
@@ -74,11 +74,15 @@ protected void SetListOffset(float value, bool smooth)
 			targetListOffset = value;
 			if (!smooth)
 			{
+				var oldListOffset = currentListOffset;
 				currentListOffset = value;
 
 				// Update mouseover
-				var mi = new MouseInput(MouseInputEvent.Move, MouseButton.None, 0, Viewport.LastMousePos, Modifiers.None, 0);
-				Ui.HandleInput(mi);
+				if (oldListOffset != currentListOffset)
```

Comment:
"Comparison of floating point numbers with equality operator. Possible loss of precision while rounding values."
Do we care about that here?